arch/arm64/boot/dts/qcom/msm8953-lenovo-kuntao.dts

```diff
@@ -63,11 +106,95 @@
 	status = "okay";
 };
 
+&i2c_3 {
+	status = "okay";
+
+	touchscreen@20 { // Broken Touch
+		compatible = "syna,rmi4-i2c";
+		reg = <0x20>;
+		interrupts-extended = <&tlmm 65 IRQ_TYPE_EDGE_FALLING>;
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&ts_reset_default &ts_int_default>;
+
+		syna,startup-delay-ms = <100>;
+
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		vdd-supply = <&pm8953_l22>;
+		vio-supply = <&pm8953_l6>;
+
+		rmi4-f01@1 {
+			reg = <0x1>;
+			syna,nosleep-mode = <1>;
+		};
+
+		rmi4-f12@12 {
+			reg = <0x12>;
+			syna,sensor-type = <1>;
+			syna,clip-x-high = <1080>;
+			syna,clip-y-high = <1920>;
+		};
+
+		rmi4-f1a@1a {
+			reg = <0x1a>;
+		};
+	};
+};
+
+&mdss {
+	status = "okay";
+};
+
+&mdss_dsi0 {
+	status = "okay";
```

Comment:
It should be the last property.

Reply:
@barni2000 when compiling this, putting on last property it is giving `Error: ../arch/arm64/boot/dts/qcom/msm8953-lenovo-kuntao.dts:137.2-18 Properties must precede subnodes
FATAL ERROR: Unable to parse input tree`